terraform/modules/enclave/README.md

```diff
@@ -52,3 +52,13 @@ To ssh to the instance, with an ssh tunnel to view the web interface (using the
     ssh ubuntu@<ip_from_output> -L 9090:localhost:9090
```

Comment:
Do we need to remove the ssh instructions here as I assume they are no longer relevant?

Reply:
i've left ssh in place for the minute while we evaluate session manager, but my hope is that this will go away in future